buildpack/telemetry/appdynamics.py

```diff
@@ -18,6 +18,38 @@
     "machine-agent",
 )
 
+APPDYNAMICS_NODE_NUMBER = os.getenv("CF_INSTANCE_INDEX", default=0)
+APPDYNAMICS_TIER = util.get_vcap_data()["application_name"]
+
+# Number suffix will be added to AppDynamics Node Name
+# The key will be used in the function '_filter_default_env'
+APPDYNAMICS_NODE_NAME_KEY = "APPDYNAMICS_AGENT_NODE_NAME"
+
+APPDYNAMICS_DEFAULT_ENV_VARS = {
```

Comment:
Lets drop the `default`
```suggestion
APPDYNAMICS_ENV_VARS = {
```

Reply:
done